gemini/use-cases/retrieval-augmented-generation/utils/intro_multimodal_rag_utils.py

```diff
@@ -596,17 +557,7 @@ def get_document_metadata(
                     stream=True,
                 )
 
-                image_embedding_with_description = (
-                    get_image_embedding_from_multimodal_embedding_model(
-                        project_id=project_id,
-                        image_uri=image_name,
-                        text=response[:text_emb_text_limit],
-                        embedding_size=embedding_size,
-                    )
-                )
-
```

Comment:
Why is this not being used anymore? This seems like the most relevant embeddings to collect